src/calendarutils.cpp

```diff
@@ -53,7 +53,8 @@ CalendarData::Event::Event(const KCalendarCore::Event &event)
     , startTime(event.dtStart())
     , endTime(event.dtEnd())
     , allDay(event.allDay())
-    , uniqueId(event.uid())
+    , uniqueId(event.instanceIdentifier())
+    , incidenceId(event.uid())
```

Comment:
This seems a bit confusing naming when uniqueId is not the same thing as uid.  First I thought incidenceId was the instance identifier.

Reply:
Ok, incidenceId is the id returned by incidence->uid(). It is only required in the dissociate case, because the dissociation and the save are separated at the moment. The dissociation is done in one place, creates a Data::Event structure, pass it to the manager which pass it back with the save command, which in turn creates a new KCalendarCore::Event from the Data::Event.